client/my-sites/site-settings/settings-discussion/main.jsx

```diff
@@ -29,7 +28,6 @@ const SiteSettingsDiscussion = ( { site, translate } ) => (
 		/>
 
 		<SiteSettingsNavigation site={ site } section="discussion" />
-		<FediverseDeprecatedDiscussionSection />
```

Comment:
This had been here since we migrated the settings from Discussion to Connections. Seemed like a good time to get rid of it.